packages/next/src/server/config-shared.ts

```diff
@@ -109,7 +116,7 @@ export interface ExperimentalTurboOptions {
    *
    * @see [Turbopack Loaders](https://nextjs.org/docs/app/api-reference/next-config-js/turbo#webpack-loaders)
    */
-  loaders?: Record<string, TurboLoaderItem[]>
+  rules?: Record<string, TurboRule>
```

Comment:
Do we want to warn/gracefully handle previous naming so we don't just break the old config?